homeassistant/components/qbus/switch.py

```diff
@@ -0,0 +1,91 @@
+"""Support for Qbus switch."""
+
+from typing import Any
+
+from qbusmqttapi.discovery import QbusMqttOutput
+from qbusmqttapi.state import QbusMqttOnOffState, StateType
+
+from homeassistant.components.mqtt import ReceiveMessage
+from homeassistant.components.switch import (
+    ENTITY_ID_FORMAT,
+    SwitchDeviceClass,
+    SwitchEntity,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .coordinator import QbusConfigEntry
+from .entity import QbusEntity
+
+PARALLEL_UPDATES = 0
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: QbusConfigEntry, add_entities: AddEntitiesCallback
+) -> None:
+    """Set up switch entities."""
+    coordinator = entry.runtime_data
+
+    # Keep a record of added entities
+    known_items: list[QbusMqttOutput] = []
```

Comment:
```suggestion
    added_items: list[QbusMqttOutput] = []
```
I think that if we call it `added_items` we can remove the comment